cdk/src/open-data-platform/data-plane/data-import/write-water-systems-data.handler.ts

```diff
@@ -85,25 +94,23 @@ function getTableRowFromRow(row: any): SqlParametersList | null {
     return null;
   }
 
-  const reportedLeadConnections = getValueOrDefault(properties.lead_connections_count);
-  let leadConnectionsCount;
+  let leadConnectionsCount = getValueOrDefault(properties.lead_connections_count);
 
   // If there is a reported lead count, use that
-  if (reportedLeadConnections > 0) {
-    leadConnectionsCount = reportedLeadConnections;
-    // Otherwise, get estimate from prediction interval.
-  } else {
+  const leadCountIsNotReported = leadConnectionsCount == null || leadConnectionsCount < 0;
+  const lowEstimate = getValueOrDefault(properties.low);
+  const highEstimate = getValueOrDefault(properties.high);
+  if (leadCountIsNotReported && lowEstimate != null && highEstimate != null) {
     // TODO: Switch to storing low, high in db.
-    leadConnectionsCount =
-      (getValueOrDefault(properties.low) + getValueOrDefault(properties.high)) / 2;
+    leadConnectionsCount = (lowEstimate + highEstimate) / 2;
```

Comment:
since we are editing this now would it be a good time to differntiate between known / predicted lead connections in db? We could add an additional column to track each or just a column which tracks isConfirmed

Reply:
Added an is_estimate column based on leadCountIsNotReported